File: implementation/scipamato/scipamato-common-wicket/src/test/java/ch/difty/scipamato/web/ModeTest.java
package ch.difty.scipamato.web;

import static org.assertj.core.api.Assertions.*;

import org.junit.Test;

public class ModeTest {

    @Test
    public void testValues() {
        assertThat(Mode.values()).containsExactly(Mode.EDIT, Mode.VIEW, Mode.SEARCH);
    }
}

File: implementation/scipamato/common/wicket/src/test/kotlin/ch/difty/scipamato/common/web/ModeTest.kt
package ch.difty.scipamato.common.web

import org.assertj.core.api.Assertions.assertThat

import org.junit.jupiter.api.Test

internal class ModeTest {

    @Test
    fun testValues() {
        assertThat(Mode.values()).containsExactly(Mode.EDIT, Mode.VIEW, Mode.SEARCH)
    }
}
